airbyte-webapp/src/pages/connections/ConnectionTransformationPage/DbtCloudTransformationsCard.tsx

```diff
@@ -39,7 +83,9 @@ export const DbtCloudTransformationsCard = ({ connection }: { connection: WebBac
   const { hasDbtIntegration, isSaving, saveJobs, dbtCloudJobs } = useDbtIntegration(connection);
 
   return hasDbtIntegration ? (
-    <DbtJobsForm saveJobs={saveJobs} isSaving={isSaving} dbtCloudJobs={dbtCloudJobs} />
+    <DbtCloudErrorBoundary>
+      <DbtJobsForm saveJobs={saveJobs} isSaving={isSaving} dbtCloudJobs={dbtCloudJobs} />
+    </DbtCloudErrorBoundary>
```

Comment:
One of the downsides of using a suspense query is that we need an error boundary! Maybe we could use a generic `ComponentErrorBoundary` or something that we can wrap individual components. That's probably better than crashing the whole page and showing `<ErrorOccurredView>`. Anyway, not for this PR, just food for thought 🙂